doc/getting_started/installation_linux.rst

```diff
@@ -10,6 +10,7 @@ applications on the following Linux distributions:
 
 * Ubuntu 16.04 LTS 64-bit
 * Fedora 25 64-bit
+* Arch Linux (install [zephyr-sdk](https://aur.archlinux.org/packages/zephyr-sdk) package from AUR)
```

Comment:
That's wiki syntax for links.  In reST it should be:

    * Arch Linux (install `zephyr-sdk <https://aur.archlinux.org/packages/zephyr-sdk>`_ package from AUR)